Modules/Dnn.CommunityForums/ActiveForums.csproj

```diff
@@ -270,7 +270,7 @@
     <EmbeddedResource Include="CustomControls\Resources\MenuButton.js" />
     <EmbeddedResource Include="CustomControls\Resources\Validation.js" />
     <EmbeddedResource Include="CustomControls\Resources\spacer.gif" />
-    <EmbeddedResource Include="sql\FullTextInstallPart2.sql" />
+    <!-- <EmbeddedResource Include="sql\FullTextInstallPart2.sql" /> -->
```

Comment:
@WillStrohl Those files do not exist inside the solution. When I tried to compile I was getting an error in the console that it could not find those files

Reply:
@ArielBlanco1990 Okay. That's fine.  Then we need to delete the lines, instead of commenting them out.

Reply:
@WillStrohl Understood. I thought about doing that at first, but wanted to check with you first.